torch/_inductor/kernel/mm_scaled.py

```diff
@@ -566,8 +552,9 @@ def tuned_scaled_mm(
                     choices,
                     input_nodes=input_nodes,
                     layout=layout,
-                    workspace_arg=get_workspace_arg(
-                        kwargs["NUM_SMS"], mat_a.get_device()
+                    workspace_arg=get_tma_workspace_arg(
+                        num_tma_descriptors=3,
```

Comment:
we actually only need 2, if you care to update as well, my top of stack had the C stores but that was buggy anyways

Reply:
Thanks for catching this! Helped me find a nasty bug in the new template code, too.